src/connector.js

```diff
@@ -69,6 +71,13 @@ let querySummaryState;
 let didYouMeanState;
 let pagerState;
 let lastCharKeyUp;
+let activeSuggestion = 0;
+let activeSuggestionWaitMouseMove = true;
```

Comment:
This variable can be declared directly inside the `updateSearchBoxState` function instead of globally.